osmosis/assetlist.json

```diff
@@ -1117,6 +1117,55 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkosmo.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkosmo.svg"
       }
+    },
+    {
+      "description": "The PAGE token is used for actions in the PageDAO NFT literary ecosystem and for DAO governance.",
+      "denom_units": [
+        {
+          "denom": "ibc/23A62409E4AD8133116C249B1FA38EED30E500A115D7B153109462CD82C1CD99",
+          "exponent": 8,
+          "aliases": [
+            "PAGE"
```

Comment:
This ("PAGE") as an alias of the Base denom is a bit confusing (although it could be used as an alias for the exp:8 denom_unit). For the base denom unit, I'd recommend aliases: 'gravity0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e' and '0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e'